### PR TITLE
BreezServices: only return currencies with known rates

### PR DESCRIPTION
**Pull request by ok300**

### CODE REVIEW

**Review by erdemyerebasmaz — APPROVED** [review]
LGTM

File: libs/sdk-core/src/fiat.rs [hunk 1]
```diff
@@ -64,8 +64,7 @@ fn convert_to_fiat_currency_with_id(id: String, info: CurrencyInfo) -> FiatCurre
 
 #[tonic::async_trait]
 impl FiatAPI for BreezServer {
-    /// Retrieve all available fiat currencies defined in a local configuration file
-    fn list_fiat_currencies(&self) -> Result<Vec<FiatCurrency>> {
+    fn list_fiat_currencies_unfiltered(&self) -> Result<Vec<FiatCurrency>> {
```
Comment (roeierez) [inline libs/sdk-core/src/fiat.rs:67]: I don't think we should rename it. From the point of view of this trait there is no filtering.

**Review by roeierez — CHANGES_REQUESTED** [review]

File: libs/sdk-core/src/fiat.rs (hunk 1 above)
Comment (ok300) [reply libs/sdk-core/src/fiat.rs:67]: I only renamed it to differentiate from the other method, which would be identically named but have a different effect.

Should I then rename the `BreezServices` method? (and all callers)

Or leave both names identical, but risk confusion when someone reads the docs?

Comment (ok300) [reply libs/sdk-core/src/fiat.rs:67]: Good point, will update the PR.

Comment (ok300) [reply libs/sdk-core/src/fiat.rs]: Done.

Ready for review again @roeierez @erdemyerebasmaz

**Review by roeierez — APPROVED** [review]
LGTM! I think we should test with c-breez wallet before merge.

**Comment by ok300** [comment]
What's a good way to build the c-breez apk with a specific breez-sdk branch?

**Pull request merged**